tests/views/api/test_permissions.py

```diff
@@ -15,11 +17,16 @@ class InstancePermissionsAPITest(TestCase):
     def setUpTestData(cls):
         cls.graph = Graph.new(
             name="INSTANCE_PERMISSIONS_TEST_GRAPH",
-            is_resource=True,
+            is_resource=False,  # creates a nodegroup, will undo this below.
```

Comment:
I adjusted this test to create a nodegroup on this test graph because the resource permissions checks return False if there are no nodegroups, and the sense of the test is that (with default allow) the anonymous user should pass.

The fact that this was failing on my first push before I adjusted the test setup is a smell related to the bug I was fixing here.